alphacsc/utils/dictionary.py

```diff
@@ -92,7 +92,7 @@ def _patch_reconstruction_error(X, z, D):
                    for diff_i in diff], axis=1)
 
 
-def get_lambda_max(X, D_hat, sample_weights=None):
+def get_lambda_max(X, D_hat, sample_weights=None, q=1):
```

Comment:
@CedricAllain can you add docstring to this function?

Reply:
I have updated, hope it is fine.